aws/logs_monitoring/lambda_function.py

```diff
@@ -4,41 +4,46 @@
 # Copyright 2021 Datadog, Inc.
 
 import json
-import os
-import boto3
 import logging
-import requests
+import os
 from hashlib import sha1
 
-from datadog_lambda.wrapper import datadog_lambda_wrapper
+import boto3
+import requests
 from datadog import api
-from enhanced_lambda_metrics import parse_and_submit_enhanced_metrics
-from steps.parsing import parse
-from steps.enrichment import enrich
-from steps.transformation import transform
-from steps.splitting import split
+from datadog_lambda.wrapper import datadog_lambda_wrapper
+
 from caching.cache_layer import CacheLayer
+from enhanced_lambda_metrics import parse_and_submit_enhanced_metrics
 from forwarder import Forwarder
 from settings import (
+    DD_ADDITIONAL_TARGET_LAMBDAS,
     DD_API_KEY,
-    DD_SKIP_SSL_VALIDATION,
     DD_API_URL,
     DD_FORWARDER_VERSION,
-    DD_ADDITIONAL_TARGET_LAMBDAS,
     DD_RETRY_KEYWORD,
+    DD_SITE,
+    DD_SKIP_SSL_VALIDATION,
 )
+from steps.enrichment import enrich
+from steps.parsing import parse
+from steps.splitting import split
+from steps.transformation import transform
 
 logger = logging.getLogger()
 logger.setLevel(logging.getLevelName(os.environ.get("DD_LOG_LEVEL", "INFO").upper()))
 
 # DD_API_KEY must be set
 if DD_API_KEY == "<YOUR_DATADOG_API_KEY>" or DD_API_KEY == "":
-    raise Exception("Missing Datadog API key")
+    raise Exception(
+        "Missing Datadog API key. Set DD_API_KEY environment variable. "
+        "See: https://docs.datadoghq.com/serverless/forwarder/"
+    )
 # Check if the API key is the correct number of characters
 if len(DD_API_KEY) != 32:
     raise Exception(
-        "The API key is not the expected length. "
-        "Please confirm that your API key is correct"
+        f"Invalid Datadog API key format. Expected 32 characters, received {len(DD_API_KEY)}. "
+        f"Verify your API key at https://app.{DD_SITE}/organization-settings/api-keys"
```

Comment:
We could use a multi line string here, and below
```suggestion
        f'''
        Invalid Datadog API key format. Expected 32 characters, received {len(DD_API_KEY)}.
        Verify your API key at https://app.{DD_SITE}/organization-settings/api-keys
        '''
```